http-server/build.gradle

```diff
@@ -26,6 +46,9 @@ jar {
 }
 
 task portableInstaller(type: Zip, group: 'release', dependsOn: shadowJar) {
+    from file('configuration-prerelease.yml')
+    from file('configuration-production.yml')
```

Comment:
This bundles the configuration with the http-server zip. When extracting we'll need to drop these in the "working directory" from where we run the dropwizard fat jar. Which configuration to use is a main method arg.

EG:
```
java -jar bin/http-server-VERSION.jar server configuration-production.yml
```

Reply:
Is this documented somewhere?

I feel like this is an easy thing to miss.

Reply:
It will be in the ansible deployment scripts. For dev/prerelease running with no-args will default to the right values.